sale_order_general_discount_triple/models/sale_order_line.py

```diff
@@ -4,51 +4,44 @@
 class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
-    discount = fields.Float(
-        string="Discount (%)",
-        digits="Discount",
-        compute=False,
-        store=True,
-        readonly=False,
-        default=0.0,
-    )
+    discount2 = fields.Float(compute="_compute_discount2", store=True, readonly=False)
+    discount3 = fields.Float(compute="_compute_discount3", store=True, readonly=False)
 
-    @api.model
-    def default_get(self, fields):
-        vals = super().default_get(fields)
-        general_discount = (
-            self.env["ir.config_parameter"]
-            .sudo()
-            .get_param(
-                "sale_order_general_discount_triple.general_discount", "discount"
-            )
-        )
-        value = vals.get("discount") or 0.0
-        vals.update(
-            {
-                "discount": 0.0,
-                "discount2": 0.0,
-                "discount3": 0.0,
-            }
-        )
+    @api.depends("product_id", "product_uom", "product_uom_qty")
+    def _compute_discount(self):
+        if hasattr(super(), "_compute_discount"):
```

Comment:
I don't think you need this if, as the compute is on the module that declares the field, isn't it?